File: backends/php-laravel/app/UseCases/Gate/Dtos/GateIssueDto.php
<?php

/**
 * This is a program developed by BobTabo.
 *
 * Copyright (c) 2026 BobTabo. All Rights Reserved.
 */

declare(strict_types=1);

namespace App\UseCases\Gate\Dtos;

use App\Support\Dtos\AbstractDto;

/**
 * JWT発行DTOクラスです。
 *
 * @author Satoshi Nagashiba <satoshi.nagashiba@gmail.com>
 * @package App\UseCases\Gate\Dtos
 */
class GateIssueDto extends AbstractDto
{
    public string $memberId = '';
    public string $accessToken = '';
}

File: backends/java-spring/src/main/java/com/authorization/usecases/gate/dtos/GateIssueDto.java
/**
 * This is a program developed by BobTabo.
 *
 * Copyright (c) 2026 BobTabo. All Rights Reserved.
 */
package com.authorization.usecases.gate.dtos;

import com.authorization.support.dtos.AbstractDto;
import lombok.Getter;
import lombok.Setter;

/**
 * JWT発行DTOクラスです。
 *
 * @author Satoshi Nagashiba <satoshi.nagashiba@gmail.com>
 */
@Getter
@Setter
public class GateIssueDto extends AbstractDto {

    private String memberId = "";
    private String accessToken = "";
}
